cmake-template.SlackBuild

```diff
@@ -115,8 +115,10 @@ cd build
     -DCMAKE_C_FLAGS:STRING="$SLKCFLAGS" \
     -DCMAKE_CXX_FLAGS:STRING="$SLKCFLAGS" \
     -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_DOCDIR="doc/$PRGNAM-$VERSION" \
+    -DCMAKE_INSTALL_INFODIR=info \
+    -DCMAKE_INSTALL_MANDIR=man \
     -DLIB_SUFFIX=${LIBDIRSUFFIX} \
-    -DMAN_INSTALL_DIR=/usr/man \
```

Comment:
This is perhaps more contentious. Sourcegraph suggests that the built-in `CMAKE_INSTALL_MANDIR` is much more widely used than `MAN_INSTALL_DIR` - [9.2k results](https://sourcegraph.com/search?q=context:global+CMAKE_INSTALL_MANDIR+file:.*CMake.*+count:100000&patternType=standard&sm=0&groupBy=repo) vs [1.5k](https://sourcegraph.com/search?q=context:global+MAN_INSTALL_DIR+file:.*CMake.*+count:100000&patternType=standard&sm=0&groupBy=repo). I'd say `MAN_INSTALL_DIR` should be defined for projects that don't use the built-in variable.